packages/plugins/embarkjs/src/index.js

```diff
@@ -46,11 +57,62 @@ class EmbarkJS {
     this.events.setCommandHandler("embarkjs:contract:generate", this.addContractArtifact.bind(this));
     this.events.setCommandHandler("embarkjs:contract:runInVm", this.runInVm.bind(this));
 
+    if (!this.enabled) return;
+
     embark.registerActionForEvent("pipeline:generateAll:before", this.addEmbarkJSArtifact.bind(this));
     embark.registerActionForEvent("pipeline:generateAll:before", this.addContractIndexArtifact.bind(this));
+
+    if (this.config.blockchainConfig.enabled) {
```

Comment:
This `if` is not needed, as it's already covered by the condition above

Reply:
Still relevant

Reply:
Agreed, not needed per https://github.com/embarklabs/embark/blob/15bd66fbaa0dcab2307c2a89738cf3a6c11dafad/packages/plugins/embarkjs/src/index.js#L25